### ISSUE TITLE
Show Options When Running "cci flow info"

### ISSUE THREAD

**Issue by Julian88Tex** [SFDO-Tooling/CumulusCI#2835]
Currently, when you run `cci flow info` referencing a flow that has a task with options you only receive feedback about the task name and not any of its options. This can create confusion when debugging a flow since you may attempt to run a task without those options and receive a different result not knowing that `cci flow info` isn't providing this feedbac.

Example Flow:
```
    foo_bar:
        steps:
            1:
                task: bar_foo
                options:
                    foo: bar
```

After running `cci flow info foo_bar` result is:
```
Flow Steps
1) task: bar_foo [from current folder]
```

I would expect something closer to:
```
Flow Steps
1) task: bar_foo [from current folder]
        options:
            foo: bar
```

Since this may result in very long results when using `cci flow info`, it may make more sense to simply use an identifier like an * to signify that tasks or flows have options so that the user knows they need to reference the cci yml before attempting to run that task/flow on it's own.

**Comment by davisagli** [comment]
We've talked about adding a `--verbose` flag to show the options. But it makes sense that it would be helpful to at least have an indicator for which tasks have options, even in the non-verbose mode.

**Comment by jstvz** [comment]
Closed by #2845
